Components/Translation/PaymentTranslator.php

```diff
@@ -28,13 +28,14 @@ public function translate(array $paymentMethod, int $languageId): array
 
         $paymentId = $paymentMethod['id'];
 
-        if (!\is_null($paymentTranslations[$paymentId]['description'])) {
+        if ($paymentTranslations[$paymentId]['description'] !== null) {
```

Comment:
Can you please use a check for the specific type of description instead of null?
What about empty strings?

Reply:
Then the shopkeeper has made a mistake or he has given an empty description. But what is the case now?